doc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -61,6 +61,9 @@
 - [ ] **Stage 3 - Community Prod Testing**
   - [ ] Documentation
     - [ ] Ensure that [README.md](https://github.com/ipfs/js-ipfs/tree/master/README.md) is up to date
+      - [ ] Install section
+      - [ ] API calls
+      - [ ] Packages Listing
```

Comment:
Are the package list & API call sections even that useful?

For a new beginner the API listing is totally overwhelming (the tutorial link is probably more useful) and for an old hand they'll go straight to the interface-core docs - maybe we could just link to it instead of having to keep all the links in sync?

Not sure what use the package listing is other than 'hey we've got loads of modules' and shout-outs to the lead maintainers.

Reply:
> For a new beginner the API listing is totally overwhelming (the tutorial link is probably more useful) and for an old hand they'll go straight to the interface-core docs - maybe we could just link to it instead of having to keep all the links in sync?

I honestly feel the same myself. I think the old API description in the README was less verbose and easier to parse. That said, the README was updated to this state by the docs effort of 2018 upon User Research. That said x2, I think it is time for our new @ipfs/docs team to assess its usefulness and adjust if necessary.

> Not sure what use the package listing is other than 'hey we've got loads of modules' and shout-outs to the lead maintainers.

Yes, it is useful:
- Here are the other repos that you might want to consider checking if you are interested in js-ipfs (vs. checking the 300 total repos)
- Look, we are putting hard work on testing, coverage and so on
- Thanks to our awesome lead maintainers for putting the effort to make this happen.

Reply:
> Are the package list & API call sections even that useful?

A good question for our stellar @ipfs/docs team! :)

Reply:
Maybe we should break the readme down into files in the `/doc` directory?  Might make it a bit easier to comprehend?

It'd be good for the @ipfs/docs team to take a look at it at any rate.  I think @ericronne and @terichadbourne are missing from that team.

Reply:
> It'd be good for the @ipfs/docs team to take a look at it at any rate. I think @ericronne and @terichadbourne are missing from that team.

They are all there https://github.com/orgs/ipfs/teams/docs/members

Reply:
> They are all there

Ah yeah, they just weren't shown on the list on the left when you click the link to the.. nevermind...